services/cad-service/app.py

```
@@ -0,0 +1,61 @@
+"""FastAPI application for CAD operations using CadQuery."""
+from __future__ import annotations
+
+from typing import Any, Dict
```

Ruff (0.14.1)

`typing.Dict` is deprecated, use `dict` instead

(UP035)